packages/data-stores/package.json

```diff
@@ -44,6 +45,7 @@
 		"wpcom-proxy-request": "^6.0.0"
 	},
 	"peerDependencies": {
+		"react": "^16.13.1",
```

Comment:
I was hoping we can avoid mixing data and components together as mentioned in the original issue:
> SVG icons can probably stay in Gutenboarding since they are used only in the <Features /> component.

A solution I see is to add a `FeatureIcon` component in Gutenboarding for now (we can move it anytime to `automattic/onboarding` package if we need it elsewhere) that takes a feature slug as prop and renders the correct icon.

Reply:
I paused for a moment with this as well. :-)

Since these are a static list of labels/slugs/icons, I think I'd expect them to be in somewhere else than data store and simply imported to plans grid. No super strong feeling about it tho, and I like Razvan's suggestion.

Data store would then be responsible of moving around feature slugs only e.g. as storing selected features.

So visual representation (icons, and I'd argue also translatable label, and descriptions) would be separate from the data (i.e. slugs).

Reply:
Yes, I saw that in the issue but I figured the plans page will be using these icons soon (p1598338944149400-slack-CPFFQTG69). But tbh, it didn't occur to me that this package is **data**-stores. I'll put them in the `automattic/onboarding` package.

Reply:
Now I see the icons next to the features in the new plans grid design 👍 
Having them in `automattic/onboarding` makes sense then!